lib/matplotlib/_tff_header_parse.py

```diff
@@ -0,0 +1,401 @@
+
```

Comment:
the lgtm PR check is flagging some encoding issue. Python 2 uses `ascii` by default and Python 3 `utf-8` so that would explain why that's not an issue for you, but in a 2 environment I think this would be a problem